cybox/test/__init__.py

```diff
@@ -52,8 +54,8 @@ def round_trip(o, output=False, list_=False):
 
     klass = o.__class__
     if output:
-        print "Class: ", klass
-        print "-" * 40
+        print("Class: ", klass)
```

Comment:
Yeah, the "output" flag is typically off; it's only there to add during development, so you can actually see the output at each stage of the round trip. Ultimately these "testing utilities" should move to mixbox, we can fix it there. Does logging output show up in test failures?